Documentation/design/renderphase.md

```diff
@@ -0,0 +1,133 @@
+# Render
+
+## Goals
+
+1. Render is not required to be run separately. This is exposed as a means of allowing users a deeper entry point for install-time customization.
+
+2. This renders all the assets necessary to bring up a cluster on to disk. This allows users to customize assets without over-parameterizing the `InstallConfig`.
+
+3. Allow user to commit the assets that describe a cluster.
+
+## Non-goals
+
+1. Render is incapable of allowing users to customize cluster objects beyond those already supported by the object's operator.
```

Comment:
nit: We could make a note that the plan is to allow for arbitrary patch objects to be provided to the installer to satisfy this requirement (but that also could just muddy this discussion with more details than necessary).

Reply:
That still remains a customization that the operator supports. And might be per operator basis...